### PR TITLE
fix-up the broken link

### PR DESCRIPTION
**Pull request by tanjunchen**
There urls are not valid.They will return 404 when clicking them. 

https://istio.io/docs/concepts/what-is-istio/overview.html ==> https://istio.io/docs/concepts/what-is-istio/

https://jobs.lever.co/openai/f163bf64-278e-417b-ad2e-5e508a29eb71 ==> https://jobs.lever.co/openai

https://github.com/containernetworking/cnihttps://github.com/containernetworking/cni ==> https://github.com/containernetworking/cni

https://thenewstack.io/beyond-ci-cd-how-continuous-hacking-of-docker-containers-and-pipeline-driven-security-keeps-ygrene-secure/index.html ==> https://thenewstack.io/beyond-ci-cd-how-continuous-hacking-of-docker-containers-and-pipeline-driven-security-keeps-ygrene-secure/

https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L210 ==> https://github.com/kubernetes/test-infra/tree/master/prow/plugins

https://helm.sh/docs/developing_charts/ ==> https://helm.sh/docs/topics/charts/

https://istio.io/docs/concepts/what-is-istio/overview.html ==> https://istio.io/docs/concepts/what-is-istio/

### CODE REVIEW

File: content/en/docs/reference/glossary/helm-chart.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Helm Chart
 id: helm-chart
 date: 2018-04-12
-full_link: https://helm.sh/docs/developing_charts/
+full_link: https://helm.sh/docs/
```
Comment (butuzov) [inline content/en/docs/reference/glossary/helm-chart.md]: Isn't it suppose to be https://helm.sh/docs/howto/charts_tips_and_tricks/ ?

Comment (tanjunchen) [reply content/en/docs/reference/glossary/helm-chart.md]: @butuzov  OK. I will update https://helm.sh/docs/topics/charts/

File: content/en/case-studies/ygrene/index.html [hunk 2]
```diff
@@ -101,7 +101,7 @@ <h2>In less than a decade, <a href="https://ygrene.com/index.html" style="text-d
...
-Cloud native also affected how Ygrene’s 50+ developers and contractors work. Adams and Arnold spent considerable time "teaching people to think distributed out of the box," says Arnold. "We ended up picking what we call the Four S’s of Shipping: safely, securely, stably, and speedily." (For more on the security piece of it, see their <a href="https://thenewstack.io/beyond-ci-cd-how-continuous-hacking-of-docker-containers-and-pipeline-driven-security-keeps-ygrene-secure/index.html">article</a> on their "continuous hacking" strategy.) As for the engineers, says Adams, "they have been able to advance as their software has advanced. I think that at the end of the day, the developers feel better about what they’re doing, and they also feel more connected to the modern software development community."<br><br>
```
Comment (butuzov) [inline content/en/case-studies/ygrene/index.html]: one more https://thenewstack.io/beyond-ci-cd-how-continuous-hacking-of-docker-containers-and-pipeline-driven-security-keeps-ygrene-secure/

Comment (tanjunchen) [reply content/en/case-studies/ygrene/index.html]: @butuzov  Thanks!

**Comment by tanjunchen** [comment]
@butuzov  done.

**Comment by butuzov** [comment]
> @butuzov done.

/lgtm

But I am not code owner (just a contributor as you), let's wait for Monday so PR can be reviewed by someone from sig-docs

**Comment by butuzov** [comment]
> Changing LGTM is restricted to collaborators
Yeap, @k8s-ci-bot, that what I have said!

**Comment by tanjunchen** [comment]
ref:https://github.com/kubernetes/website/issues/17649

**Pull request merged**